Source/Parallelization/GuardCellManager.cpp

```diff
@@ -110,12 +103,18 @@ guardCellManager::Init(
         // the stencil of the FFT solver. Here, this number (`ngFFT`)
         // is determined *empirically* to be the order of the solver
         // for nodal, and half the order of the solver for staggered.
-        IntVect ngFFT;
-        if (do_nodal) {
-            ngFFT = IntVect(AMREX_D_DECL(nox_fft, noy_fft, noz_fft));
-        } else {
-            ngFFT = IntVect(AMREX_D_DECL(nox_fft/2, noy_fft/2, noz_fft/2));
-        }
+
+        int ngFFt_x = do_nodal ? nox_fft : nox_fft/2.;
+        int ngFFt_y = do_nodal ? noy_fft : noy_fft/2.;
+        int ngFFt_z = do_nodal ? noz_fft : noz_fft/2.;
+
+        ParmParse pp;
+        pp.query("nx_guard_psatd", ngFFt_x);
+        pp.query("ny_guard_psatd", ngFFt_y);
+        pp.query("nz_guard_psatd", ngFFt_z);
```

Comment:
Assuming the above change of parameter name, this would become:
```
        ParmParse pp("psatd");
        pp.query("nx_guard", ngFFt_x);
        pp.query("ny_guard", ngFFt_y);
        pp.query("nz_guard", ngFFt_z);
```

Reply:
Done in commit 4fcbac6900123363f58dc8.